src/components/NewCommentForm.tsx

```diff
@@ -73,20 +142,36 @@ export const NewCommentForm: React.FC = () => {
         <div className="control">
           <textarea
             id="comment-body"
-            name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={cn('textarea', { 'is-danger': touchedBody })}
+            value={body}
+            onChange={event => setBody(event.target.value)}
+            onBlur={() => setTouchedBody(true)}
           />
         </div>
 
-        <p className="help is-danger" data-cy="ErrorMessage">
-          Enter some text
-        </p>
+        {
+          touchedBody && (
+            <>
+              {!body && (
+                <p className="help is-danger" data-cy="BodyField">
+                  Enter some text
+                </p>
+              )}
+            </>
+          )
+        }
```

Comment:
merge those conditions
```suggestion
          touchedBody &&!body && (
                <p className="help is-danger" data-cy="BodyField">
                  Enter some text
                </p>
          )
        }
```